io.github.fheroes2.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.fheroes2</id>
+  <launchable type="desktop-id">io.github.fheroes2.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <translation type="gettext">fheroes2_Main</translation>
+  <name>fheroes2</name>
+  <developer_name>fheroes2</developer_name>
+  <summary>Free Heroes of Might and Magic II (fheroes2) is a recreation of HoMM2 game engine.</summary>
+  <description>
+    <p>Free Heroes of Might and Magic II (fheroes2) is a recreation of HoMM2 game engine.</p>
+    <p>This open source multiplatform project, written from scratch, is designed to reproduce the original game with significant improvements in the gameplay, graphics and logic (including support for high-resolution graphics, improved AI, numerous fixes and UI improvements), breathing new life into one of the most addictive turn-based strategies.</p>
+    <p>To play fheroes2 you will need to copy the 'ANIM', 'DATA', 'MAPS' and 'MUSIC' folders from Heroes II to '~/.var/app/io.github.fheroes2/data/fheroes2</p>
+  </description>
+  <url type="homepage">https://ihhub.github.io/fheroes2/</url>
+  <url type="bugtracker">https://github.com/ihhub/fheroes2/issues</url>
+  <releases>
+    <release date="2022-05-07" version="v0.9.15"/>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/ihhub/fheroes2/master/docs/images/screenshots/screenshot_world_map.png</image>
```

Comment:
please link to an actual release or revision, because otherwise when this file will get changed, the build will fail

Reply:
Done